migrate.sh

```diff
@@ -57,6 +65,10 @@ function ha::running() {
 			)"
 			[[ -n "${id}" ]] && return 0
 			return 1
+			;;
+		haos)
+			status="$(ha core stats)"
```

Comment:
I would prefer to just pipe the output of `ha core stats` to `/dev/null` instead of creating an unused variable. This should be fine:
```suggestion
            ha core stats > /dev/null 2>&1
            return
```
A simple `return` is enough to return the status of the last executed command. No need to use `$?` here.

Reply:
Yeath, that could be done also, Don't know why I did it like this.. :)